src/server/config/db.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+import mongoose from 'mongoose'
+
+var uri = 'mongodb://localhost/yoda'
+var options = {
+    server: {
+        poolSize: 20        // amount of connection in one instance
```

Comment:
To be absolutely anal, if you're putting comments inline, there should be precisely 2 spaces after the code and before the start of the comment. So it becomes:

       poolSize: 20  // amount of connection in one instance. 

That way the code looks cleaner overall. I'm not sure if this was mentioned in the styleguide. :-)